### DISCUSSION TITLE
Mesa 22.3.0

### DISCUSSION THREAD

**Discussion by Directox01** [pal1000/mesa-dist-win#120] [General]
Hello @pal1000 may I ask if already beta version exist for mesa 22.3.0?

**Comment by pal1000** [comment]
Not yet. First release candidate should be out [a week from today](https://docs.mesa3d.org/release-calendar.html). 

### Changes in 22.3.0
Beside background bug fixes and improvements:
- Gallium raw test libraries `graw.dll` and `graw_null.dll` have been removed.
- OpenGL over D3D12 standalone ICD `openglon12.dll` has been removed. System-wide deployment users would be migrated to gallium megadriver `libgallium_wgl.dll` which still includes OpenGL over D3D12 driver.
- VA-API over D3D12 driver debut.

**Reply by Directox01** [reply]
Sound very good. Thank you for the link too!

Have a wonderful day!

**Comment by Directox01** [comment]
@pal1000 You don't mind if I ask if u have information if there is chance to have opengl 4.4 support till end of this year with mesa?

Thank you.

**Reply by pal1000** [reply]
If you are talking about d3d12 driver I don't know, [but you can ask for it](https://gitlab.freedesktop.org/mesa/mesa/-/issues).
